src/storage/v2/indices/vector_index.hpp

```diff
@@ -43,21 +68,25 @@ struct VectorIndexInfo {
 /// This structure includes the index name, the label and property on which the index is created,
 /// and the configuration options for the index in the form of a JSON object.
 struct VectorIndexSpec {
+  // TODO(@DavIvek): Add scalar kind configuration options
```

Comment:
append: "... because it's only useful if/when we add better support for quantization."